mathesar/rpc/roles.py

```diff
@@ -53,6 +54,10 @@ class RoleInfo(TypedDict):
     description: Optional[str]
     members: Optional[list[RoleMember]]
 
+    @classmethod
+    def from_dict(cls, d):
+        return cls
+
```

Comment:
This is broken. You need to instantiate the `cls` on `d`. Given that, you don't actually need this method. If you don't want validation (fair when returning info from the DB), you can just use `RoleInfo(role)` where you call this.